react/features/overlay/functions.js

```diff
@@ -19,6 +24,10 @@ export function getOverlayToRender(state: Object) {
         }
     }
 
+    if (loadConfigOverlayVisible) {
```

Comment:
Let's drop this please. This screen must be disconnected from the "overlays" feature. We need to navigate to connecting as soon as we have a URL we are going to open.